packages/jokul/src/components/message/types.ts

```diff
@@ -1,18 +1,13 @@
-import type { AriaRole } from "react";
-import type { Density, WithChildren } from "../../core/types.js";
+import type { Density } from "../../core/types.js";
 
-export interface MessageProps extends WithChildren {
-    id?: string;
-    title?: string;
+export interface MessageProps extends React.HTMLAttributes<HTMLDivElement> {
```

Comment:
Jeg lurer på om `ComponentPropsWithoutRef` Er hakket mer komplett enn `HTMLAttributes`, men begge får jobben gjort 😊
```suggestion
export interface MessageProps extends React.ComponentPropsWithoutRef<"div"> {
```